### PR TITLE
[FLINK-9152] Harmonize BroadcastProcessFunction Context names

### PR DESCRIPTION
**Pull request by aljoscha**
R: @kl0u

### CODE REVIEW

File: flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/co/CoBroadcastWithKeyedOperator.java [hunk 1]
```diff
@@ -33,6 +33,8 @@
 import org.apache.flink.streaming.api.TimeDomain;
 import org.apache.flink.streaming.api.TimerService;
 import org.apache.flink.streaming.api.functions.co.KeyedBroadcastProcessFunction;
+import org.apache.flink.streaming.api.functions.co.KeyedBroadcastProcessFunction.Context;
```
Comment (kl0u) [inline flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/co/CoBroadcastWithKeyedOperator.java]: Unused import.

File: flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/co/KeyedBroadcastProcessFunction.java [hunk 2]
```diff
@@ -89,11 +89,11 @@
 	 * query the current processing/event time, and also query and update the internal
 	 * {@link org.apache.flink.api.common.state.BroadcastState broadcast state}. In addition, it
 	 * can register a {@link KeyedStateFunction function} to be applied to all keyed states on
-	 * the local partition. These can be done through the provided {@link Context}.
+	 * the local partition. These can be done through the provided {@link BaseBroadcastProcessFunction.Context}.
```
Comment (kl0u) [inline flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/co/KeyedBroadcastProcessFunction.java]: Remove the `BaseBroadcastProcessFunction`.

File: flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/co/KeyedBroadcastProcessFunction.java [hunk 3]
```diff
@@ -89,11 +89,11 @@
...
 	 * The context is only valid during the invocation of this method, do not store it.
 	 *
 	 * @param value The stream element.
-	 * @param ctx A {@link Context} that allows querying the timestamp of the element,
+	 * @param ctx A {@link BaseBroadcastProcessFunction.Context} that allows querying the timestamp of the element,
```
Comment (kl0u) [inline flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/co/KeyedBroadcastProcessFunction.java]: Remove the `BaseBroadcastProcessFunction`.

**Comment by aljoscha** [comment]
Thanks for the review! I merged

**Pull request closed without merging**